install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# Build the docker image
+docker build -t os-exploder .
+
+# start kubernetes service account
+kubectl create -f os-exploder-sa.yaml
+
+# start k8s pod with the api token for the above service account
+kubectl create -f os-exploder-pod.yaml
```

Comment:
Will this create the pod under the previously created serviceaccount, or is that handled in the pod yml?